.changeset/honest-olives-kneel.md

```diff
@@ -0,0 +1,6 @@
+---
+"@hashicorp/flight-icons": minor
+---
+
+Icons added: accessibility, channel, minus-circle-fill, plus-circle-fill  
+Icons changed: plus-circle
```

Comment:
I went ahead and added a line break for you so the text doesn't run together.

Reply:
Thank you!